### DISCUSSION TITLE
Having Error deploying smart contract on sepolia testnet using Remix

### DISCUSSION THREAD

**Discussion by J4Web** [Cyfrin/foundry-full-course-cu#2525] [Q&A]
I was trying to deploy my solidity smart contract on sepolia testnet but the deploy button for it is disabled 
I'm providing the code below for reference and an image of what's happening in the Remix.

Code 

```
// SPDX-License-Identifier: MIT
pragma solidity ^0.8.18;

contract SimpleStorage {
    uint256 public favNum = 69;

    struct Person {
        string name;
        uint256 age;
    }

    function modifyFavNum(uint256 _num) public {
        favNum = _num;
    }

    function showNum() view public returns(uint256) {
        return favNum;
    }

    Person public jay = Person({
        name: "Hero",
        age: 26
    });

    Person[] public personArray;

    mapping(string => uint256) public dictionary;

    function addUserToArray(string memory _name, uint256 _age) public {
        Person memory newPerson = Person(_name, _age);
        personArray.push(newPerson);
        dictionary[_name] = _age;
    }
}


```

<img width="1375" alt="Screenshot 2024-09-22 at 3 37 48 AM" src="https://github.com/user-attachments/assets/472738e2-9678-40ee-babe-3d318dbf694c">

**Comment by EngrPips** [comment]
Hello @J4Web, I tried deploying the contract you shared on my end, and it works fine can you maybe refresh the browser and retry?

<img width="1440" alt="Screenshot 2024-09-22 at 03 30 40" src="https://github.com/user-attachments/assets/98fd8fa8-f7c8-4775-bd02-f5f21942e5cc">

**Comment by mumernisar** [comment]
@J4Web Account is not selected ..!!
If using metamask maybe not properly not configured..!!

**Comment by Morwena0** [comment]
had the same issue and tried to deploy to arbitrum sepolia and its working

**Reply by EngrPips** [reply]
Interesting.

**Comment by mumernisar** [comment]
> Are you getting the metamask authorization window when deploying to arbitrum and that your account is connected to remix..!!!?  The only issue visible is your account not showing..!! Which can certainly lead to deploy button disabling.

**Reply by Morwena0** [reply]
im pretty sure that there was not any issue with metamask, checked many times

**Reply by mumernisar** [reply]
@Morwena0 then can't say anything friend..!! 
Maybe clear data for the site and try again.
Other than that .. good luck🤞🏻
